gst/tensor_filter/tensor_filter.h

```diff
@@ -128,7 +128,7 @@ struct _GstTensor_Filter_Framework
        * @return the size of input tensors
        */
   int (*getOutputDimension) (const GstTensor_Filter * filter,
-      void **private_data, tensor_dim outputDimension, tensor_type * type);
+      void **private_data, GstTensor_TensorsMeta * meta);
```

Comment:
setInputDimension is not changed accordingly.

Reply:
It's not considered yet. This commit only covers `nnstreamer_filter_example`. It will be updated later.

Reply:
yes I will :)